trunner/device.py

```diff
@@ -18,6 +18,7 @@
 
 
 _BOOT_DIR = PHRTOS_PROJECT_DIR / '_boot'
+LOGFILE_PATH = '/actions-runner/logfile'
```

Comment:
this is another change which prevents using `trunner` locally!

**to be removed ASAP!**
(as this current PR is a stop-gap, leave it as-is for now, but start thinking how to fix that in the near future)

Please also note that the file would be removed with the container (eg. after docker image update)

Reply:
ok, let's leave it for now. Side remark: `logfile` is a too-generic name (this is not a generic log, You're logging only reboot timeouts here)